### DISCUSSION TITLE
Background issue of holes

### DISCUSSION THREAD

**Discussion by chiba1sonny** [wkentaro/labelme#1055] [Q&A / Help] [answered]
Thanks for the great work.
I have a question about the annotation of holes.
If I annotated a hole as "_background_", and converted the JSON data to Voc data, is this hole(labeled "_background_") and actual background(unlabelled) is same?

![image](https://user-images.githubusercontent.com/68838083/169656490-9d4c2021-d6c6-4765-9b69-7818976da7fe.png)
like in this image, are part 1 and part 2 the same?

**Comment by wkentaro** [comment, marked as answer]
You need to introduce an additional class.
In the example, `__ignore__` class is there in addition to `_background_`.
https://github.com/wkentaro/labelme/blob/main/examples/semantic_segmentation/labels.txt
In your case, I guess the hole can be categorized as either `__ignore__` or `hole`.

**Reply by chiba1sonny** [reply]
Thanks for your kind reply.
